3rdparty/openssl/CMakeLists.txt

```diff
@@ -292,7 +292,7 @@ ExternalProject_Add(
   INSTALL_COMMAND "")
 
 add_dependencies(openssl_includes openssl_configure)
-add_dependencies(openssl_generated openssl_configure)
+add_dependencies(openssl_generated openssl_includes)
 add_dependencies(openssl_generated openssl_copied)
```

Comment:
There seems to be an ordering bug here. `openssl_copied` does not depend on anything else and therefore could be launched before openssl_configure.

Reply:
It should be fine given that `openssl_copied` and `openssl_configure` are independent.  `opensslcrypto` depends on `openssl_generated`, which ensures both  `openssl_copied` and `openssl_configure` will be ready before building the library.

Reply:
From the code, `openssl_configure` creates `${OPENSSL_BUILD_INCLUDES}` and hence implicitly `${OPENSSL_BUILD}`
https://github.com/openenclave/openenclave/blob/b8451393224fa0ea18cd0607983c6ef4f4519ff9/3rdparty/openssl/CMakeLists.txt#L77

`openssl_copied` write to `${OPENSSL_BUILD}`
https://github.com/openenclave/openenclave/blob/b8451393224fa0ea18cd0607983c6ef4f4519ff9/3rdparty/openssl/CMakeLists.txt#L285-L286


Therefore there is a dependency.

Reply:
Fixed